compiler-rt/lib/asan/asan_interceptors.cpp

```diff
@@ -584,6 +584,12 @@ INTERCEPTOR(char *, strcpy, char *to, const char *from) {
 INTERCEPTOR(char*, strdup, const char *s) {
   void *ctx;
   ASAN_INTERCEPTOR_ENTER(ctx, strdup);
+#  if SANITIZER_WINDOWS
+  // Allowing null input is Windows-specific
+  if (UNLIKELY(!s)) {
```

Comment:
nit: for consistency, please don't use `{ }` for single statement bodies.

nit: `if (SANITIZER_WINDOWS && UNLIKELY(!s))`